test/test_iperf.ml

```diff
@@ -194,42 +194,42 @@ let test_tcp_iperf_two_stacks_basic amt timeout () =
   let module Test = Test_iperf (Vnetif_backends.Basic) in
   Test.default_network () >>= fun { backend; Test.client; Test.server } ->
   Test.V.record_pcap backend
-    (Printf.sprintf "tests/pcap/tcp_iperf_two_stacks_basic_%d.pcap" amt)
+    (Printf.sprintf "tcp_iperf_two_stacks_basic_%d.pcap" amt)
     (Test.tcp_iperf ~server ~client amt timeout)
 
 let test_tcp_iperf_two_stacks_mtu amt timeout () =
   let module Test = Test_iperf (Vnetif_backends.Mtu_enforced) in
   Test.default_network () >>= fun { backend; Test.client; Test.server } ->
   Test.V.record_pcap backend
-    (Printf.sprintf "tests/pcap/tcp_iperf_two_stacks_mtu_%d.pcap" amt)
+    (Printf.sprintf "tcp_iperf_two_stacks_mtu_%d.pcap" amt)
```

Comment:
pcap recording still grabs the full trace, as far as I know.  We should probably rig up a proper command-line argument for recording these traces (and allow the user to specify a directory there too).  @samoht how hard would that be to do with alcotest?